_wallets/cobowallet.md

```diff
@@ -0,0 +1,47 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+id: Cobo
+title: "Cobo Wallet"
+titleshort: "Cobo"
+compat: "mobile android ios"
+level: 3
+platform:
+  - mobile:
+    name: mobile
+    os:
+      - name: android
+        text: "walletcobo"
+        link: "https://play.google.com/store/apps/details?id=cobo.wallet"
+        source: "https://github.com/cobowallet"
```

Comment:
This should be a direct link to the repository containing the wallet source code, instead of a link to the profile page for the organization's GitHub account.

Reply:
If we are not an open source wallet, is that ok? So I can't provide a link to the source code.

Reply:
> This should be a direct link to the repository containing the wallet source code, instead of a link to the profile page for the organization's GitHub account.

If we are not an open source wallet, is that ok? So I can't provide a link to the source code.

Reply:
Open (public) source is a requirement for listing.  You can see a full list of the requirements [here](https://github.com/bitcoin-dot-org/bitcoin.org/blob/master/docs/managing-wallets.md).